doc/config.yml

```
@@ -33,11 +33,26 @@ Extensions:
         name: FENIX
         repo: https://github.com/idaholab/fenix
         menu:
-            Documentation and Resources:
-                Complete Code Manual: syntax/index.md
-                Code Standards: sqa/fenix_scs.md
-                Contributing to FENIX: /sqa/contributing.md
+            Getting Started:
+                Installation: getting_started/installation.md
+                FENIX on HPC: getting_started/hpc.md
+                Using FENIX: getting_started/using_fenix.md
+                User Notes: getting_started/user_notes.md
+                Contributing to FENIX: sqa/contributing.md
+            Documentation:
+                FENIX-only Syntax: syntax/fenix_only.md
+                Complete Code Syntax: syntax/index.md
+                V&V and Examples: /verification_validation_examples/index.md
+                Publications using FENIX: publications.md
+            Software Quality:
                 Software Quality Assurance (SQA) Information: /sqa/index.md
+                Code Standards: sqa/fenix_scs.md
+                Code Coverage: https://mooseframework.inl.gov/fenix/docs/coverage
```

```suggestion
                #Code Coverage: https://mooseframework.inl.gov/fenix/docs/coverage
```

I took a peek at the CIVET scripts, and it seems (at first glance....could have missed something late yesterday) that the current HTML generation script for coverage assumes the sync URL based on the location of the repository rather than whether it is `private` or not. I am suggesting we comment this out for now and then restore it when we make the site fully public. 

I have made an issue (#35). 

This should be the last item before we merge this - I think this version of the website looks good!